ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -636,6 +637,8 @@ getCompletions plId ideOpts CC {allModNamesAsNS, anyQualCompls, unqualCompls, qu
         , enteredQual `T.isPrefixOf` original label
         ]
 
+      getModuleName line = let ws = filter (/= "qualified") (T.words line)
```

Comment:
A comment explaining why this works would be good. Also, I don't really understand why we're doing sort-of-parsing here, don't we have a `ParsedModule` lying around?

Reply:
`Maybe` we do. I added cases for when we can use the `ParsedModule` to get the module name. :slightly_smiling_face: